### PR TITLE
fix: Spec and Doc fixes for Release 1.3.1

### PR DESCRIPTION
**Pull request by kevinshahfws**
This PR contains below fixes:

1. Create enum values in BooleanMap for device.hdr Response
2. Create enum values in BooleanMap for device.hdcp Response
3. Create enum values in BooleanMap for device.audio Response
4. Update device.videoResolution description and result
5. Update Device.screenResolution description and result
6. Update Accessibility.closedCaptionsSettings and Accessibility.closedCaptions key options
7. Update Accessibility.voiceGuidance and Accessibility.voiceGuidanceSettings key options
8. Update Advertising.appBundleId example
9. Update Advertising.advertisingId to define attributes and updated description
10. Update Advertising.policy capability to singular capability
11. Update description for Device.version

### CODE REVIEW

File: src/openrpc/device.json [hunk 1]
```diff
@@ -473,7 +446,7 @@
     },
     {
       "name": "screenResolution",
-      "summary": "Get the current screen resolution",
+      "summary": "Get the maximum supported video resolution for the graphical surface of the app. \n\nThe pairs returned will be of a [width, height] format and will correspond to the following values: \n\nNTSC Standard Definition (SD): [720, 480] \n\nPAL Standard Definition (SD): [720, 576] \n\nHigh Definition (HD): [1280, 720] \n\nFull HD (FHD): [1920, 1080]\n\n4K Ultra High Definition (UHD): [1920, 1080] \n\n**Note:** This method will be deprecated with a future Firebolt release",
```
Comment (kpears201) [inline src/openrpc/device.json:449]: This should not say video resolution here

File: src/openrpc/device.json [hunk 2]
```diff
@@ -793,16 +834,108 @@
       },
       "AudioProfiles": {
         "title": "AudioProfiles",
```
Comment (kpears201) [inline src/openrpc/device.json:836]: Does this need "type": "object"?

File: src/openrpc/device.json [hunk 3]
```diff
@@ -793,16 +834,108 @@
...
-            "propertyNames": {
-              "$ref": "https://meta.comcast.com/firebolt/types#/definitions/AudioProfile"
-            }
+          "dolbyDigital5.1": {
+            "type": "boolean"
+          },
+          "dolbyDigital5.1+": {
+            "type": "boolean"
+          },
+          "dolbyAtmos": {
```
Comment (kpears201) [inline src/openrpc/device.json:847]: Should all these properties be required?

File: src/openrpc/device.json [hunk 4]
```diff
@@ -793,16 +834,108 @@
...
+            "type": "boolean"
+          }
+        }
+      },
+      "HDRType": {
```
Comment (kpears201) [inline src/openrpc/device.json:852]: Wonder if HDRFormat is a better name?

Comment (kpears201) [reply src/openrpc/device.json:852]: https://en.wikipedia.org/wiki/High-dynamic-range_television#Formats

File: src/openrpc/device.json [hunk 5]
```diff
@@ -793,16 +834,108 @@
...
+        },
+        "required": [
+          "hdr10",
+          "hdr10Plus",
+          "dolbyVision",
+          "hlg"
+        ],
+        "description": "The type of HDR that is supported"
+      },
+      "HDCPType": {
```
Comment (kpears201) [inline src/openrpc/device.json:877]: Maybe HDCPVersion?

File: src/openrpc/device.json [hunk 6]
```diff
@@ -793,16 +834,108 @@
...
+        "title": "DeviceVersion",
+        "type": "object",
+        "properties": {
+          "sdk": {
+            "$ref": "https://meta.comcast.com/firebolt/types#/definitions/SemanticVersion",
+            "description": "The Firebolt SDK version"
+          },
+          "api": {
+            "$ref": "https://meta.comcast.com/firebolt/types#/definitions/SemanticVersion",
+            "description": "The lateset Firebolt API version supported by the curent device."
```
Comment (kpears201) [inline src/openrpc/device.json:904]: typo curent -> current

Comment (kpears201) [reply src/openrpc/device.json:904]: typo lateset -> latest

Comment (kpears201) [reply src/openrpc/device.json:904]: you just copied it over though, guess these typos always been there!

File: src/schemas/types.json [hunk 7]
```diff
@@ -36,9 +36,7 @@
             "enum": [
```
Comment (kpears201) [inline src/schemas/types.json:36]: Can't we just remove AudioProfile all together? I don't think it is used anymore

File: src/openrpc/advertising.json [hunk 8]
```diff
@@ -404,6 +389,27 @@
...
+        "properties": {
+          "ifa": {
+            "type": "string",
+            "description": "UUID conforming to IAB standard"
+          },
+          "ifa_type": {
+            "type": "string",
+            "description": "source of the IFA as defined by IAB"
+          },
+          "lmt": {
```
Comment (kpears201) [inline src/openrpc/advertising.json:405]: add enum so lmt can only be 0 or 1

File: src/schemas/types.json (hunk 7 above)
Comment (kevinshahfws) [reply src/schemas/types.json:36]: Entertainment schema uses AudioProfile schema from Types

File: src/openrpc/device.json [hunk 9]
```diff
@@ -793,16 +834,115 @@
...
+          },
+          "hdcp2.2": {
+            "type": "boolean"
+          }
+        },
+        "required": [
+          "hdcp1.4",
+          "hdcp2.2"
+        ],
+        "description": "The type of HDCP that is supported"
```
Comment (kpears201) [inline src/openrpc/device.json:899]: did we want to change type to version here too?

File: src/openrpc/device.json [hunk 10]
```diff
@@ -793,16 +834,115 @@
...
+            "type": "boolean"
           }
+        },
+        "required": [
+          "hdr10",
+          "hdr10Plus",
+          "dolbyVision",
+          "hlg"
+        ],
+        "description": "The type of HDR that is supported"
```
Comment (kpears201) [inline src/openrpc/device.json:882]: did we want to change type to format here too?

**Review by kpears201 — DISMISSED** [review]

**Review by alkalinecoffee — DISMISSED** [review]

File: src/openrpc/device.json [hunk 11]
```diff
@@ -793,16 +834,115 @@
...
+      },
+      "HDCPVersion": {
```
Comment (alkalinecoffee) [inline src/openrpc/device.json:884]: Please change this to `HDCPVersionMap` to differentiate it from a future `HDCPVersion` object.

Same for `HDRFormat` to `HDRFormatMap`.

**Review by kpears201 — APPROVED** [review]

**Review by alkalinecoffee — APPROVED** [review]

**Pull request merged**